### DISCUSSION TITLE
How do I connect containers to gluetun on the Synology GUI?

### DISCUSSION THREAD

**Discussion by Jgreene44** [qdm12/gluetun#903] [Q&A] [answered]
I have built the container in the GUI and now I have the following containers. 
![image](https://user-images.githubusercontent.com/24441757/159184474-b4595277-3aec-4125-8753-eff4713ddcf2.png)
The gluetun container is using PIA to connect, I have input my username and password and when starting the container the logs say it is running successfully. Now I need to connect my other 3 containers to the gluetun container but am unsure of how. All of the documentation that I have found is doing it by editing the config but in the Synology GUI I don't have access to any of that (that I know of). 

This is what the synology gui looks like for the containers. 
![image](https://user-images.githubusercontent.com/24441757/159184580-4732712e-4fac-427b-9fb1-4f152e57b65b.png)
![image](https://user-images.githubusercontent.com/24441757/159184588-e608ba79-e773-4c67-8946-20827b052e7b.png)
![image](https://user-images.githubusercontent.com/24441757/159184595-2bdd1d16-5ae8-4991-939a-6682f3152915.png)
![image](https://user-images.githubusercontent.com/24441757/159184608-a4a4a450-8b5d-4e6c-b6b7-63a3f00a98c7.png)
![image](https://user-images.githubusercontent.com/24441757/159184634-95b8cc7a-b163-4139-af3b-425798dd15bd.png)

The photos above is the GUI for the other types of containers but the gluetun one is similar with just different file path and environmental variables. 

There is a network tab in the main portion of the docker GUI but there aren't any options to make the gluetun container my main connection. I have looked into linking containers some but am not sure if that is the path to go down. My end goal is to get the other three containers to use the gluetun container as the network connection, any help would be greatly appreciated.

The synology is am using is DS1821+ and the version of software I am running is DSM 7.0.1-42218 Update 2.

**Comment by qdm12** [comment, marked as answer]
Hey there! You may be able to do it with `Links`, although these are meant to be deprecated I think. If you link one of your containers to gluetun it may use it, but I'm not 100% sure either.

Otherwise, I would recommend you to setup a Portainer container which is a web GUI for Docker, and you can setup gluetun and connect containers through it. Sadly the Synology GUI has definitely limitations. You could also try `docker-compose` with `docker-compose.yml` files, it's quite useful and easier than it looks like at a first glance, although it's all in the terminal/SSH.

Good luck!!

**Reply by Jgreene44** [reply]
I appreciate the feedback! I think I will try to get Portainer setup on the Synology NAS, because I'd really like everything to be self contained on the NAS.
